src/main/java/PostgresCommunicator.java

```diff
@@ -46,6 +46,21 @@ public void updateOpportunityNumericField(Opportunity opportunity, String column
         executeQuery(sqlQuery);
     }
 
+    public void deleteOpportunity(Opportunity opportunity) throws SQLException, ClassNotFoundException {
+        String uuid = opportunity.getUuid();
+        String sqlQuery = String.format("DELETE FROM opportunities WHERE uuid = '%s';", uuid);
+        executeQuery(sqlQuery);
+
+    }
+
+    public boolean rowExists(String uuid) throws SQLException, ClassNotFoundException {
+        Connection db = getConnection();
+        String sqlQuery = String.format("SELECT name FROM opportunities WHERE uuid = '%s';", uuid);
+        Statement stmt = db.createStatement();
+        ResultSet rs = stmt.executeQuery(sqlQuery);
```

Comment:
I'd avoid the shortened names in here for readability - `stmt` could be `statement`, `rs` could be `resultSet`